client/my-sites/subscribers/subscriber-list/subscriber-list.tsx

```diff
@@ -37,7 +36,6 @@ export const SubscriberList = ( { subscribers, locale }: SubscriberListProps ) =
 					subscription_id,
 					display_name,
 					email_address,
-					subscriptions,
 					openRate,
```

Comment:
Just a heads up: in my upcoming PR I've renamed openRate to open_rate because that's probably how it's going to come in from the API later on (they use snake case 😑).